span.go

```diff
@@ -381,7 +399,15 @@ func (s *Span) reportSelfTime() {
 	} else {
 		s.tx.childrenTimer.childEnded(endTime)
 	}
-	s.tx.spanTimings.add(s.Type, s.Subtype, s.Duration-s.childrenTimer.finalDuration(endTime))
+
+	// Collect span timings when dropped and the destination service isn't empty
+	spanDuration := s.Duration - s.childrenTimer.finalDuration(endTime)
```

Comment:
```suggestion
	selfTime := s.Duration - s.childrenTimer.finalDuration(endTime)
```

This isn't the entire span duration, but the "self time": the span duration minus the overlap with child spans. Is that the correct duration to be accumulating for dropped spans as well as breakdown metrics?

Reply:
I'm not sure what I did here, I'll just use the `s.Duration` since that accurately reflects the span duration, and revert the childrenTimer calculation to what it was set to.